Sources/Utils/Backtrace.cpp

```diff
@@ -67,13 +69,76 @@ void PrintBacktrace() {
   }
 }
 #elif defined(OS_WIN32)
+
+static HANDLE CurrentProcess() {
```

Comment:
This is confusing as a name as it is very similar to `GetCurrentProcess`.  Can we rename this to make it more obvious that this is something else?  How about something like `SymGetCurrentProcess`?  That identifies that this is something for symbolication.

Reply:
Done.